build.sbt

```diff
@@ -49,12 +49,12 @@ lazy val commonSettings = Seq(
   javaOptions in (Test, run) ++= Seq("-Xms64m", "-Xmx64m"),
   libraryDependencies ++= Seq(
     compilerPlugin("org.typelevel" %% "kind-projector" % "0.10.0"),
-    "org.typelevel" %%% "cats-core" % "1.6.0",
-    "org.typelevel" %%% "cats-laws" % "1.6.0" % "test",
-    "org.typelevel" %%% "cats-effect" % "1.3.0",
-    "org.typelevel" %%% "cats-effect-laws" % "1.3.0" % "test",
-    "org.scala-lang.modules" %%% "scala-collection-compat" % "0.3.0",
-    "org.scalatest" %%% "scalatest" % "3.0.6" % "test"
+    "org.typelevel" %%% "cats-core" % "2.0.0-M1",
+    "org.typelevel" %%% "cats-laws" % "2.0.0-M1" % "test",
+    "org.typelevel" %%% "cats-effect" % "2.0.0-M1",
+    "org.typelevel" %%% "cats-effect-laws" % "2.0.0-M1" % "test",
+    "org.scala-lang.modules" %%% "scala-collection-compat" % "1.0.0",
```

Comment:
I think 1.0.0 was a bad build and we're supposed to stick with 0.3.0 until 2.0.0. From the README for that project:
> Note: Please do not release any artifacts against version 1.0.0. Due to #195 we are planning on releasing 2.0.0 that is neither source nor binary compatible with 1.0.0.

Reply:
Oh. I didn't realize that. For sure it makes sense to wait for 2.0.0.

Also, these changes should be binary compatible for fs2, right? It doesn't define any testing modules that'd depend on the problematic dependencies of cats. (or does it)

Reply:
#1454 removed the scala-collection-compat dependency altogether. I agree with your reasoning on why the cats M1 build is okay.